src/App.tsx

```diff
@@ -65,6 +66,8 @@ function App() {
   const { isAuthenticated, isLoading, getIdTokenClaims } = useAuth0();
 
   const [notifications, setNotifications] = useState<Notification[]>([]);
+  const [isFormVisible, setIsFormVisible] = useState(true);
+  const isMobileOrTablet = useMediaQuery('(max-width: 1024px)');
```

Comment:
```suggestion
  const isSmallViewport = useMediaQuery('(max-width: 1024px)');
```
I think this is clearer, the key point here is viewport, not device (i.e. if someone uses a desktop monitor in portrait mode, it could also trigger ...)